README.md

```diff
@@ -3,7 +3,9 @@ Spotify for Nuvola Player 3
 
 Integration of Spotify into your linux desktop via
 [Nuvola Player](https://github.com/tiliado/nuvolaplayer).
- 
+
+Based on the Nuvola Player 2 Spotify plugin by Chase Colman (<chase@colman.io>).
```

Comment:
How much is your work based on Chase's original script? If there are significant code portions from his script, you have to retain copyright from the original file.